app/filtering.js

```diff
@@ -582,6 +586,8 @@ function registerPermissionHandler (session, partition) {
       } else if (isFullscreen && alwaysAllowFullscreen) {
         // Always allow fullscreen if setting is ON
         response.push(true)
+      } else if (isOpenExternal && isDeniedScheme.indexOf(protocol.toLowerCase()) >= 0) {
+        response.push(false)
```

Comment:
could you move this to the https://github.com/brave/browser-laptop/pull/15260/files#diff-8a9bac13ef6264feb2b6da7c18d86b3cR562 to make the diff smaller

Reply:
actually this code block wouldn't be reached otherwise since it's an `else if`